src/renderer/components/MessageWrapper.js

```diff
@@ -216,10 +216,13 @@ function convert (message) {
 
   message.onDelete = () => ipcRenderer.send('deleteMessage', message.id)
 
-  var msg = message.msg
+  const msg = message.msg
 
-  if (msg.isSetupmessage) msg.text = window.translate('autocrypt_asm_click_body')
-  msg = Object.assign(msg, {
+  if (msg.isSetupmessage) {
+    msg.text = window.translate('autocrypt_asm_click_body')
+  }
+
+  Object.assign(msg, {
```

Comment:
No need to assign `msg` since `Object.assign()` mutates first argument, so `msg` can be const.